### DISCUSSION TITLE
同一条http隧道绑定两个不同的，都备案过的域名，一个正常使用，另一个被阻断

### DISCUSSION THREAD

**Discussion by MengStar-L** [natfrp/wiki#291] [使用问题] [answered]
同一条http隧道绑定两个不同的，都备案过的域名，两域名记录解析相同。一个正常使用，另一个被阻断，为什么，怎么解决，有没有大佬救一下~被阻断的提示页面如下
尊敬的用户您好：
您访问的网站被机房安全管理系统拦截，有可能是以下原因造成：
1.您的网站未备案，或者原备案号被取消.
2.域名在访问黑名单中.
3.网站存在非法内容.

1和3肯定不是了，因为另一个域名可以访问，2怎么解决呀

[a comment by an automated account is not shown]

**Reply by MengStar-L** [reply]
好的，多谢啦

**Comment by maimai977977** [comment]
请问一下dns是多少 后台只显示了一个地址  dnspad 需要2个dns 地址才能解析
